versioned_docs/version-2.0/tutorials/Developers/agents.mdx

```diff
@@ -0,0 +1,507 @@
+---
+sidebar_position: 6
+---
+
+# Agent Evaluations
+
+We will walk through the evaluation workflow for RAG (retrieval augmented generation).
```

Comment:
Agent, not RAG. Also discuss the agent we are building, eg the SQL agent. Explain the tools we are giving it